src/components/SpecialResolution/SpecialResolutionForm.vue

```diff
@@ -309,20 +311,22 @@ export default class SpecialResolutionForm extends Mixins(DateMixin) {
       ...this.getcreateResolution,
       resolutionDate: val
     })
+    this.validate()
   }
   /** called to add new resolutionDateText. */
   protected onResolutionTextChanged (val: string) {
     this.setResolution({
       ...this.getcreateResolution,
       resolutionText: val
     })
+    this.validate()
   }
 
   /** Validation rule for name */
   private nameRules (label, isRequired = true): Array<Function> {
     return [
       v => isRequired ? (!!v?.trim() || `${label} is required`) : true,
-      v => (v?.length <= 30) || 'Cannot exceed 30 characters' // maximum character count
+      v => v?.trim() ? (v?.length <= 30) || 'Cannot exceed 30 characters' : true // maximum character count
```

Comment:
v.trim() removes spaces before/after input, but space is a character long. The schema will likely not pass this.